### ISSUE TITLE
Seed Data / General Comments

### ISSUE THREAD

**Issue by chf2** [balloosnafoo/no-kicker#7]
One thing that stands out to me for this project is that it's potentially going to require a ton of seed data. I would imagine that to demo well you should try and make it seem like you're halfway through a season (though this is ultimately up to you) -- seeding the player data table alone will be a huge task that right now doesn't have time allocated for it. I would look into ways to automate this, potentially using real historic data. I might look into ways to import tabular csv or xls data into a postgres DB.

Also make sure to use bootstrap along the way to reduce the amount of time needed to polish CSS at the end. Lastly, if you have to prioritize, I think getting the trades and lineups working well should take precedence over detailed player info views since they're more core to the functionality and since that adds an even heavier burden in terms of seed data.

**Comment by balloosnafoo** [comment]
Yeah, I was also a bit concerned about the seed data. Another thing that I am kind of curious about is whether I should avoid reference to actual players and teams? I don't imagine the NFL is waiting to squash my portfolio piece for copyright infringement but I thought it's worth asking.

**Comment by chf2** [comment]
Our general stance on things like this (likeness/copyright) is to not worry too much about it. I think especially with something like players / player data that is so publicly available you won't run into any issues. If you wanted for some reason to change names, I would still get the data from official sources and then use Faker to gsub out the names with new ones.

**Comment by chf2** [comment]
One more quick note: not so sure how helpful this is for historical data, but potentially check out http://developer.sportradar.us/. They have a 90 day free trial for their API and it's very powerful (I used it for some goofy project before App Academy started). I wouldn't spend more than an hour or two trying to figure it out though... at some point using box scores from a site like http://www.pro-football-reference.com/ might be easier

**Comment by balloosnafoo** [comment]
Oh ok thanks for the tip :) There is a python package that has historical/weekly updating data pulled from nfl.com that I'm already pretty familiar with, so I'll probably use that to minimize time spend familiarizing myself with apis. Should work as long as I don't need live data, which I wasn't planning on having.
